plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestingMySqlServer.java

```diff
@@ -46,7 +51,7 @@ public TestingMySqlServer(String dockerImageName, boolean globalTransactionEnabl
             container = container.withCommand("--gtid-mode=ON", "--enforce-gtid-consistency=ON");
         }
         this.container = container;
-        container.start();
+        reusableContainer = startOrReuse(container);
```

Comment:
the name is misleading.
it's either container or -- in case of reuse -- it's nothing (not a container, not a reference to a container).

call it `cleanup`, `close`, or make optional

Reply:
it was about "reusableContainer" name